api/resources/v1alpha1/types.go

```diff
@@ -30,6 +30,13 @@ const (
 	// true then the controller will not delete the object in case it is removed from the ManagedResource or the
 	// ManagedResource itself is deleted.
 	KeepObject = "resources.gardener.cloud/keep-object"
+	// Mode is a constant for an annotation on a resource managed by a ManagedResource. It indicates the
+	// mode that should be used to reconcile the resource.
+	Mode = "resources.gardener.cloud/mode"
+	// IgnoreMode is a constant for the value of the mode annotation describing an ignore mode.
+	// Reconciliation in ignore more removes the resource from the ManagedResource status and does not
+	// perform any action on the cluster.
+	IgnoreMode = "Ignore"
```

Comment:
I would vote for lowercase for more consistency. Any strong reason for capitalizing it?
```suggestion
	IgnoreMode = "ignore"
```

Reply:
I initially started with lowercase but then changed to uppercase because the examples from https://github.com/gardener/gardener-resource-manager/issues/82 are in uppercase. Then I also saw https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/addon-manager where the label values are also in uppercase.
I don't have a strong opinion actually. If we agree to have the annotation values in lowercase, then we should also update #82 with this detail.

Reply:
I just thought, lowercase would be more consistent with most other label/annotation values that we have in gardener, e.g. https://github.com/gardener/gardener/blob/9df08ae3ae865f96db4161eb4ad01ced73107f3e/pkg/operation/common/types.go#L226-L230 (and all other consts in this file).
But also don't have a strong opinion, you can choose.